src/.vitepress/mermaid/plugin.ts

```diff
@@ -0,0 +1,32 @@
+// Mermaidを読み込むためのVitePressプラグイン。
+//
+// # 大まかな仕組み
+// - VitePressのMarkdownレンダラーを拡張して、MermaidのコードブロックをMermaidDiagram要素に変換する。
+// - VitePressのテーマでMermaidDiagramコンポーネントを登録しておく（.vitepress/theme/index.tsを参照）。
```

Comment:
こうでも通じそう（特殊な独自記法を消して一般的な書式にリファクタリングしてる感じ）
```suggestion
// VitePressのMarkdownレンダラーを拡張して、MermaidのコードブロックをMermaidDiagram要素に変換する。
// VitePressのテーマでMermaidDiagramコンポーネントを登録しておく（.vitepress/theme/index.tsを参照）。
```